src/blocks/blocks/form/style.scss

```diff
@@ -283,6 +289,41 @@
 			margin-bottom: 0px;
 			font-size: var(--help-font-size);
 		}
+
+		@media (min-width: 640px) {
+			flex-grow: 1;
+
+			&:is(.is-style-o-c-three-quarters, .is-style-o-c-two-thirds, .is-style-o-c-half, .is-style-o-c-one-third, .is-style-o-c-one-quarter) {
+				box-sizing: border-box;
+				margin-left: 0px;
+				margin-right: 0px;
+			}
+	
+			&.is-style-o-c-three-quarters {
+				flex-basis: calc( 75% - var( --inputs-gap ) );
+				max-width: 75%;
+			}
+	
+			&.is-style-o-c-two-thirds {
+				flex-basis: calc( 66.66666666666666% - var( --inputs-gap ) );
+				max-width: 66.66666666666666%;
+			}
+	
+			&.is-style-o-c-half {
+				flex-basis: calc( 50% - var( --inputs-gap ) );
+				max-width: 50%;
+			}
+		
+			&.is-style-o-c-one-third {
+				flex-basis: calc( 33.33333333333333% - var( --inputs-gap ) );
```

Comment:
I think we can write this as `calc(100% / 3 - var( --inputs-gap ))`. Same for `66.66666%` as `calc(200% / 3)` to make it look cleaner.

Reply:
Not precisely, because if you search for `33%` you have to do the math whenever you look at it.  Also, you can not have a code search like VSCode Find because instead of searching for a value with `33%` you have to remember that is written as a calculation.

Unless the value is dynamic and we change the dividing unit, plain value work the best.